sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -188,16 +189,12 @@ rom_error_t mask_rom_boot(void) {
     if (true) {
       // Unlock execution of ROM_EXT executable code (text) sections.
       RETURN_IF_ERROR(epmp_state_check(&epmp));
-      epmp_region_t rom_ext_text;
-      RETURN_IF_ERROR(manifest_code_region_get(manifest, &rom_ext_text));
-      mask_rom_epmp_unlock_rom_ext_rx(&epmp, rom_ext_text);
+      mask_rom_epmp_unlock_rom_ext_rx(&epmp,
```

Comment:
would it make sense to add a redundant `manifest_check` here?

Reply:
Probably, but let's continue this conversation in the next PR that I'll create shortly (moves `manifest_check()` into another function).